client/app/telecom/telephony/billingAccount/billing/tollfreeHistory/telecom-telephony-billing-account-billing-tollfree-history.controller.js

```diff
@@ -102,26 +72,15 @@ angular.module("managerApp").controller("TelecomTelephonyBillingAccountBillingTo
     =            INITIALIZATION            =
     ======================================*/
 
-    function init () {
-        self.loading.init = true;
-
+    this.$onInit = function () {
         return TelephonyMediator.getGroup($stateParams.billingAccount).then(function (group) {
             self.group = group;
-
             return fetchHistory().then(function (consumptions) {
-                self.consumption.raw = consumptions;
-                self.sortConsumption();
+                self.consumptionData = consumptions;
             });
-        }).catch(function (error) {
-            Toast.error([$translate.instant("telephony_group_billing_tollfree_history_init_error"), (error.data && error.data.message) || ""].join(" "));
-            return $q.reject(error);
-        }).finally(function () {
-            self.loading.init = false;
+        }, function (err) {
+            return new ToastError([$translate.instant("telephony_group_billing_tollfree_history_init_error"), (err.data && err.data.message) || ""].join(" "));
```

Comment:
This is not how `ToastError` works. It takes as arguments an error object AND a translation key, not a string